libgalois/include/katana/PropertyGraph.h

```diff
@@ -539,6 +539,16 @@ class KATANA_EXPORT PropertyGraph {
     return edge_type_set_id_to_type_names_.at(edge_type_set_id);
   }
 
+  /// \returns a pointer to the underlying memory of the Node type ID array
+  TypeSetID* GetNodeTypeSetIDArrayData() const {
+    return node_type_set_id_.data();
+  }
+
+  /// \returns a pointer to the underlying memory of the Edge type ID array
+  TypeSetID* GetEdgeTypeSetIDArrayData() const {
+    return edge_type_set_id_.data();
+  }
```

Comment:
Would normally prefer just a const reference to `edge_type_set_id`.

Reply:
@GioLomia

Reply:
I actually prefer not returning references to internal data structures, such as `NUMAArray` to prevent accidental misuse.

Reply:
Agree with Tyler